service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceHandler.java

```diff
@@ -206,6 +206,12 @@ public void sessionOpened(IoSession session) {
             final DefaultHttpSession acceptSession = (DefaultHttpSession) session;
             // final Subject subject = ((IoSessionEx) acceptSession).getSubject();
 
+            // log warning first time we see Http 1.0 request
+            if (acceptSession.getVersion().toString().equals("HTTP/1.0")){
+                LOGGER.warn(String.format("http.proxy service %s received an HTTP 1.0 request. HTTP 1.0 is not"
```

Comment:
@vmaraloiu, this would log it every time you see it correct?  it would be better to log it just the first time